fdbserver/workloads/ReportConflictingKeys.actor.cpp

```diff
@@ -0,0 +1,166 @@
+#include "fdbclient/NativeAPI.actor.h"
```

Comment:
This file probably needs the Apache 2 license header thing at the top, and please apply clang-format to it.

Reply:
Will do. Thanks.